docs/walkthrough/choose-backbone.md

```diff
@@ -65,46 +65,43 @@ To get a list of supported models:
+│                              RN50x64::openai │ text-to-image │       1024 │  transformer │                        Open CLIP "RN50x64::openai" model │
+│                                RN50::yfcc15m │ text-to-image │       1024 │  transformer │                          Open CLIP "RN50::yfcc15m" model │
+│                      ViT-B-16::laion400m_e31 │ text-to-image │        512 │  transformer │                Open CLIP "ViT-B-16::laion400m_e31" model │
+│                      ViT-B-16::laion400m_e32 │ text-to-image │        512 │  transformer │                Open CLIP "ViT-B-16::laion400m_e32" model │
+│                             ViT-B-16::openai │ text-to-image │        512 │  transformer │                       Open CLIP "ViT-B-16::openai" model │
+│             ViT-B-16-plus-240::laion400m_e31 │ text-to-image │        640 │  transformer │       Open CLIP "ViT-B-16-plus-240::laion400m_e31" model │
+│             ViT-B-16-plus-240::laion400m_e32 │ text-to-image │        640 │  transformer │       Open CLIP "ViT-B-16-plus-240::laion400m_e32" model │
+│                        ViT-B-32::laion2b_e16 │ text-to-image │        512 │  transformer │                  Open CLIP "ViT-B-32::laion2b_e16" model │
+│                      ViT-B-32::laion400m_e31 │ text-to-image │        512 │  transformer │                Open CLIP "ViT-B-32::laion400m_e31" model │
+│                      ViT-B-32::laion400m_e32 │ text-to-image │        512 │  transformer │                Open CLIP "ViT-B-32::laion400m_e32" model │
+│                             ViT-B-32::openai │ text-to-image │        512 │  transformer │                       Open CLIP "ViT-B-32::openai" model │
+│            ViT-B-32-quickgelu::laion400m_e31 │ text-to-image │        512 │  transformer │      Open CLIP "ViT-B-32-quickgelu::laion400m_e31" model │
+│            ViT-B-32-quickgelu::laion400m_e32 │ text-to-image │        512 │  transformer │      Open CLIP "ViT-B-32-quickgelu::laion400m_e32" model │
+│                   ViT-B-32-quickgelu::openai │ text-to-image │        512 │  transformer │             Open CLIP "ViT-B-32-quickgelu::openai" model │
+│                         ViT-L-14-336::openai │ text-to-image │        768 │  transformer │                   Open CLIP "ViT-L-14-336::openai" model │
+│                             ViT-L-14::openai │ text-to-image │        768 │  transformer │                       Open CLIP "ViT-L-14::openai" model │
+│ xlm-roberta-base-ViT-B-32::laion5b_s13b_b90k │ text-to-image │        512 │  transformer │ Open MCLIP "xlm-roberta-base-ViT-B-32::laion5b_s13b_b90k"│
+│                                              │               │            │              │                                                    model │
+└──────────────────────────────────────────────┴───────────────┴────────────┴──────────────┴───────────────────━━━━━━━━━━━━━━─────────────────────────┘
```

Comment:
not related to this PR, i believe we need to remove some of the CLIP backbones, such as resnet, no one is using them.